includes/admin/class-extra-checkout-fields-for-brazil-order.php

```diff
@@ -224,6 +224,13 @@ public function order_data_after_billing_address( $order ) {
 		// Get plugin settings.
 		$settings    = get_option( 'wcbcf_settings' );
 		$person_type = intval( $settings['person_type'] );
+		$phone_label = __( 'Phone', 'woocommerce-extra-checkout-fields-for-brazil' );
+		$has_cellphone = version_compare( WC_VERSION, '3.0', '<' ) ? ! empty( $order->billing_cellphone ) : '' !== $order->get_meta( '_billing_cellphone' );
```

Comment:
Suportando da versão 5 pra frente agora. Então esta tranquilo.

```suggestion
		$has_cellphone = '' !== $order->get_meta( '_billing_cellphone' );
```